### PR TITLE
chore: add TF 1.5 examples for lambda sqs and lambda sns

### PR DESCRIPTION
**Pull request by antoinePB**
For https://albelli-photobox.atlassian.net/browse/CLD-1651

Tested both examples with terraform 1.5

### CODE REVIEW

File: examples/aws/1.5/lambda_function_sqs/main.tf [hunk 1]
```diff
@@ -0,0 +1,18 @@
+module "lambda_sqs" {
+  source = "../../../../modules/aws/lambda_function_sqs"
+
+  name       = "${var.project_name}LambdaSqs"
```
[the remark this answers is not in the document]
Comment (antoinePB) [reply examples/aws/1.5/lambda_function_sqs/main.tf:4]: I took this convention from existing examples

File: examples/aws/1.5/lambda_function_sqs/main.tf [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+}
+
+module "lambda_sqs_policy" {
+  source = "../../../../modules/aws/lambda_function_sqs"
+
+  name             = "${var.project_name}LambdaSqsPolicy"
+  source_dir       = "${path.module}/src"
+  sqs_queue_policy = templatefile("${path.module}/templates/queue_policy.json", {})
+
+  tags = var.tags
```
[the remark this answers is not in the document]
Comment (antoinePB) [reply examples/aws/1.5/lambda_function_sqs/main.tf:17]: This is taken from existing examples, so the variable is passed when calling the `examples` script (that in turn uses locals)

File: examples/aws/1.5/lambda_function_sns/variables.tf [hunk 3]
```diff
@@ -0,0 +1,4 @@
+variable "project_name" {}
+variable "tags" {
+  type = map
```
[the remark this answers is not in the document]
Comment (antoinePB) [reply examples/aws/1.5/lambda_function_sns/variables.tf:3]: There are at the root of `1.5` folder, along the examples script that calls those examples

File: examples/aws/1.5/lambda_function_sns/main.tf [hunk 4]
```diff
@@ -0,0 +1,8 @@
+module "lambda_sns" {
```
[the remark this answers is not in the document]
Comment (antoinePB) [reply examples/aws/1.5/lambda_function_sns/main.tf:1]: Good spot, I will change it

File: examples/aws/1.5/lambda_function_sqs/main.tf [hunk 5]
```diff
@@ -0,0 +1,18 @@
+module "lambda_sqs" {
+  source = "../../../../modules/aws/lambda_function_sqs"
+
+  name       = "${var.project_name}LambdaSqs"
+  source_dir = "${path.module}/src"
+
+  tags = var.tags
+}
+
+module "lambda_sqs_policy" {
```
[the remark this answers is not in the document]
Comment (antoinePB) [reply examples/aws/1.5/lambda_function_sqs/main.tf]: This is a different source, SNS vs SQS, and while they are both based on lambda_function modules, there are extra modules created for each

**Review by danibeca — APPROVED** [review]
All looks good. I don't see changes in the module itself, so we are good if the examples passed.

**Review by mayur7436 — APPROVED** [review]

**Pull request merged**